src/main/resources/db/migration/V5__alter_behandling_og_sekvens_vedtak.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE behandling
+    DROP COLUMN skalBehandlesAutomatisk;
```

Comment:
```suggestion
    DROP COLUMN skal_behandles_automatisk;
```
Kræsjer per nå da det ikke er noen kolonne som heter det